### ISSUE TITLE
Compilations

### ISSUE THREAD

**Issue by braoult** [antoinepirlot/Satunes#606]
In #603, left screenshot, the 3 first lines (`African Music - Benio`) are part of a compilation, but shown as different albums in Satunes (maybe because of the "Artist" field, different in each song ?).

It becomes highly impractical.

My compilations are tagged with the iTunes specific `TCMP` frame, recognized by various players. But there is also another different common way to differentiate compilations: Using `Various Artists` in "Album Artist" tag. This is usually done in `TPE2`, which unfortunately is pretty important for classical music, but do-able.

What is the correct way to declare compilations, so they are seen correctly in Album view ?

EDIT: This is a common and usual question, not specific to Satunes: https://www.reddit.com/r/navidrome/comments/kjarq0/dealing_with_various_artists/

**Comment by antoinepirlot** [comment]
> In [#603](https://github.com/antoinepirlot/Satunes/issues/603), left screenshot, the 3 first lines (`African Music - Benio`) are part of a compilation, but shown as different albums in Satunes (maybe because of the "Artist" field, different in each song ?).
> 
> It becomes highly impractical.
> 
> My compilations are tagged with the iTunes specific `TCMP` frame, recognized by various players. But There also a different common way to differentiate compilations: Using `Various Artists` in "Album Artist" tag. This is usually done in `TPE2`, which unfortunately is pretty important for classical music, but do-able.
> 
> What is the correct way to declare compilations, so they are seen correctly in Album view ?
> 
> EDIT: This is a common and usual question, not specific to Satunes: https://www.reddit.com/r/navidrome/comments/kjarq0/dealing_with_various_artists/

Hey, yeah, at this time, if the same album name has 2 different artists name it will be 2 different albums.

I'm also experiencing this problem.

I planned to check the standards about this case.

Thanks for the link

**Comment by antoinepirlot** [comment]
Hello @braoult , 

I modified how Satunes loads albums compilation.
I would be sure I did the right choice.

Satunes now loads the music's artist and the album's artist separately.

Here's an example:
![Image](https://github.com/user-attachments/assets/479c95d0-39cc-4cc4-8a84-cd443568f484)

On the image, the album contains musics with different names. Satunes will add this album to "Armin Van Buuren" artist and to each musics' artist. Also, the album's musics won't be added to "Armin Van Buuren" but only in album's musics' artists.

Also, if the album is marked as compilation and the album artist is unknown, then Satunes will add the album to the "Various Artists" as album artist.

These images show you the before/after result:
![Image](https://github.com/user-attachments/assets/d69c10fa-a41e-4b5a-a75c-0d823bf42d7f)
![Image](https://github.com/user-attachments/assets/a04dde1a-07d6-4b5d-a7ba-8b681c88123b)

I hope this is what you ask :)
